interactor/src/mvx_esdt_safe/mvx_esdt_safe_interactor_main.rs

```diff
@@ -114,82 +114,78 @@ impl MvxEsdtSafeInteract {
-    //     let token_nonce = 0u64;
-    //     let token_amount = BigUint::<StaticApi>::from(0u128);
-    //
-    //     let to = bech32::decode("");
-    //
-    //     let response = self
-    //         .interactor
-    //         .tx()
-    //         .from(&self.wallet_address)
-    //         .to(self.state.current_address())
-    //         .gas(30_000_000u64)
-    //         .typed(MvxEsdtSafeProxy)
-    //         .deposit(to, OptionalValue::None)
-    //         .payment((
-    //             TokenIdentifier::from(token_id.as_str()),
-    //             token_nonce,
-    //             token_amount,
-    //         ))
-    //         .returns(ReturnsResultUnmanaged)
-    //         .run()
-    //         .await;
-    //
-    //     println!("Result: {response:?}");
-    // }
-
-    // pub async fn execute_operations(&mut self) {
-    //     let hash_of_hashes = ManagedBuffer::new_from_bytes(&b""[..]);
-    //     let operation = Operation::<StaticApi>::new();
-    //
-    //     let response = self
-    //         .interactor
-    //         .tx()
-    //         .from(&self.wallet_address)
-    //         .to(self.state.current_address())
-    //         .gas(30_000_000u64)
-    //         .typed(MvxEsdtSafeProxy)
-    //         .execute_operations(hash_of_hashes, operation)
-    //         .returns(ReturnsResultUnmanaged)
-    //         .run()
-    //         .await;
-    //
-    //     println!("Result: {response:?}");
-    // }
-
-    // pub async fn register_token(&mut self) {
-    //     let egld_amount = BigUint::<StaticApi>::from(0u128);
-    //
-    //     let sov_token_id = TokenIdentifier::from_esdt_bytes(&b""[..]);
-    //     let token_type = EsdtTokenType::<StaticApi>::default();
-    //     let token_display_name = ManagedBuffer::new_from_bytes(&b""[..]);
-    //     let token_ticker = ManagedBuffer::new_from_bytes(&b""[..]);
-    //     let num_decimals = 0u32;
-    //
-    //     let response = self
-    //         .interactor
-    //         .tx()
-    //         .from(&self.wallet_address)
-    //         .to(self.state.current_address())
-    //         .gas(30_000_000u64)
-    //         .typed(MvxEsdtSafeProxy)
-    //         .register_token(
-    //             sov_token_id,
-    //             token_type,
-    //             token_display_name,
-    //             token_ticker,
-    //             num_decimals,
-    //         )
-    //         .egld(egld_amount)
-    //         .returns(ReturnsResultUnmanaged)
-    //         .run()
-    //         .await;
-    //
-    //     println!("Result: {response:?}");
-    // }
+    pub async fn deposit(
+        &mut self,
+        to: Address,
+        opt_transfer_data: OptionalValueTransferDataTuple<StaticApi>,
+        payments: PaymentsVec<StaticApi>,
+    ) {
+        let response = self
+            .interactor
+            .tx()
+            .from(&self.wallet_address)
+            .to(self.state.current_address())
+            .gas(30_000_000u64)
```

Comment:
A good habit is to save numbers as constants, especially if you are using the same one multiple times.

Put it inside a constant names GAS_LIMIT or something